pkg/printers/printers.go

```diff
@@ -33,30 +33,6 @@ func GetStandardPrinter(typer runtime.ObjectTyper, encoder runtime.Encoder, deco
 	var printer ResourcePrinter
 	switch format {
 
-	case "json", "yaml":
-		jsonYamlFlags := NewJSONYamlPrintFlags()
-		p, matched, err := jsonYamlFlags.ToPrinter(format)
```

Comment:
@soltysh @deads2k moved all printflags to `pkg/printers`.
Adding the dependency on the new `pkg/printers/flags` package here caused an import cycle.

Reply:
See https://github.com/juanvallejo/kubernetes/pull/5 for what I've meant. I've also included a few minor code changes I'd like to see :)

Reply:
(`pkg/kubectl/printers` depends on `pkg/printers/internalversion` because we use that pkg to add printHandlers in the `ToPrinter` method of humanreadable_flags.go)